crates/extensions/c8y_mapper_ext/src/json.rs

```diff
@@ -30,28 +31,21 @@ pub enum CumulocityJsonError {
 }
 
 /// Converts from thin-edge measurement JSON to C8Y measurement JSON
-pub fn from_thin_edge_json(input: &str) -> Result<String, CumulocityJsonError> {
-    let timestamp = WallClock.now();
-    let c8y_vec = from_thin_edge_json_with_timestamp(input, timestamp, None)?;
-    Ok(c8y_vec)
-}
-
-/// Converts from thin-edge Json to c8y_json with child id information
-pub fn from_thin_edge_json_with_child(
+pub fn from_thin_edge_json(
     input: &str,
-    child_id: &str,
+    entity: &EntityMetadata,
 ) -> Result<String, CumulocityJsonError> {
```

Comment:
The changes in this file highlights how the new EntityStore API can simplify and make more robust the mappers.